### PR TITLE
Issue #58 | Add Unit Tests for checkorder.js

### LINKED ISSUE #58: Add unit-tests for checkorder.js
## About
_checkorder.js_ is a script that parse the links of a .md file and check if the list is alphabetically sorted!
## Usage
```node checkorder.js LIST.md```

> To celebrate the Hacktoberfest, everyone is welcome to close this issue =)
Happy Hacktoberfest and happy hacking! 🎉

### PR DESCRIPTION
**Pull request by dgautsch**
This adds unit tests with the Jest test framework. I used node 8.11.x to build this solution. Fixes #58 

### Further Testing
- We could test `isFileOrdered` and `compareArr` but would then have to export them.

### CODE REVIEW

**Comment by dgautsch** [comment]
@lnfnunes We'll probably need to change the codacy rules for unit tests, so this passes.

File: test/checkorder.spec.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+describe('checkorder', () => {
+    beforeEach(() => {
+        jest.spyOn(process, 'exit').mockImplementationOnce(() => true);
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should idenitfy ordered list links as ordered', () => {
```
Comment (lnfnunes) [inline test/checkorder.spec.js]: Typo `idenitfy`

Comment (dgautsch) [reply test/checkorder.spec.js]: Woops! 🙂

I'll fix that up today.

**Comment by dgautsch** [comment]
@lnfnunes ok, fixed the typo :)

**Comment by dgautsch** [comment]
@lnfnunes any other changes for this?

**Comment by lnfnunes** [comment]
Thank you so much @dgautsch

**Pull request merged**